app/views/renewal/amls_turnover.scala.html

```diff
@@ -22,7 +22,7 @@
 
 @headingText = @{
     businessTypes match {
-        case Some(types) if types.length == 1 => "renewal.turnover.title.single.service"
+        case Some(types) if types.length == 1 => "renewal.turnover.title"
         case _ => "renewal.turnover.title"
     }
```

Comment:
before it was implemented in a way, that you would see different title based on number of services, now it's showing the same title in both cases, is that right?